index.html

```diff
@@ -117,5 +117,40 @@ <h3>how we started</h3>
         </ul>
       </div>
     </section>
+    <section class="contact">
+      <h2>contact</h2>
+      <form>
+        <label for="name">name</label>
+        <input type="text" id="name">
+        <label for="email">e-mail</label>
+        <input type="email" id="email">
+        <label for="topic">topic</label>
+        <textarea id="topic"></textarea>
+      </form>
+      <img src="assets/office.jpg" alt="Hello agency headquaters">
+      <div>
+        <span>Our office is located in the heart of Warsaw</span>
+        <span>Feel free to say hello!</span>
+        <div itemprop="address" itemscope itemtype="http://schema.org/PostalAddress">
+          <span itemprop="streetAddress">Warsaw St. 23/21</span>
+          <span itemprop="postalCode">03-323</span>
+          <span itemprop="addressLocality">Warsaw</span>
+          <span itemprop="addressCountry">Poland</span>
+        </div>
+      </div>
+    </section>
+    <footer>
+      <!-- assets/socialmedia.svg as a background image for footer -->
+      <ul>
+        <li><a href="https://pinterest.com"><img src="assets/pinterest.svg" aria-hidden="true"></a></li>
+        <li><a href="https://twitter.com"><img src="assets/twitter.svg" aria-hidden="true"></a></li>
+        <li><a href="https://facebook.com"><img src="assets/facebook.svg" aria-hidden="true"></a></li>
+        <li><a href="https://instagram.com"><img src="assets/instagram.svg" aria-hidden="true"></a></li>
+      </ul>
+      <div >
+        <span>designed with love by <span itemprop="name">The Awwwwesomes</span></span>
```

Comment:
Would sth like that be okay?

Reply:
Hm now I see that we don't really specify the `itemscope itemtype="http://schema.org/Organization"` in the surrounding element on our website. I think if you add it to your surrounding div it should be fine. Maybe also the "Organization" schema works better with the address above than the "Place"?

Reply:
Yes!